pages/k8s/_site/charm-aws-iam.html

```diff
@@ -0,0 +1,31 @@
+<h1 id="aws-iam-charm">AWS IAM Charm</h1>
+
+<p>The AWS IAM charm allows a Kubernetes cluster to be authenticated via the Amazon
+API using AWS users and groups. The authorization falls to RBAC, so an Amazon
+user or group maps to an RBAC user.</p>
+
+<h2 id="usage">Usage</h2>
+
+<p>The AWS IAM charm is subordinate to the [<code class="highlighter-rouge">kubernetes-master</code>]
```

Comment:
Link is broken (change to `<a href="https://jaas.ai/u/containers/kubernetes-master">kubernetes-master</a>`).